src/Microsoft.DotNet.ProjectModel/Graph/LockFileReader.cs

```diff
@@ -17,7 +18,17 @@ public static class LockFileReader
     {
         public static LockFile Read(string lockFilePath)
```

Comment:
I'd prefer not having retry code in ProjectModel at all, this could be done by helper method easily.